components/sideMenu/index.less

```diff
@@ -19,14 +19,14 @@
     .primary-menu {
         position: relative;
         transform: translateX(-100%);
-        background-color: @turquoise;
-        color: @white;
+        background-color: #ebeff7;
```

Comment:
Does it make sense to make this color (and `#cedee9` below) variables? Or do you think this is the only place they'll be used?